ddtrace/opentracer/tracer.py

```diff
@@ -3,6 +3,7 @@
 from opentracing import Format
 from opentracing.scope_managers import ThreadLocalScopeManager
 
+import ddtrace
```

Comment:
nothing important and it's fine to keep it, but in general I prefer to have all the `import`s at the top of the module instead between `from` imports. It doesn't change anything, it's just cosmetics.